sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/yandex.yaml

```diff
@@ -0,0 +1,50 @@
+provider:
+  metadata:
+    name: Yandex
+    category: Cloud provider
+
+  rules:
+    - id: yandex_oauth_token
+      rspecKey: S6760
+      metadata:
+        name: Yandex OAuth tokens should not be disclosed
+        message: Make sure this Yandex OAuth token gets revoked, changed, and removed from the code.
+      detection:
+        pre:
+          include:
+            content:
+              - y0_
+              - y1_
+              - y2_
+              - y3_
+        matching:
+          pattern: "\\b(y[0-3]_[\\w-]{55})\\b"
+      examples:
+        - text: |
+            Authorization: Bearer y3_Vdheub7w9bIut67GHeL345gfb5GAnd3dZnf08FRbvjeUFvetYiohGvc
+          containsSecret: true
+          match: y3_Vdheub7w9bIut67GHeL345gfb5GAnd3dZnf08FRbvjeUFvetYiohGvc
+        - text: |
+            py3_suffix
+          containsSecret: false
+
+    - id: yandex_iam_token
+      rspecKey: S6760
+      metadata:
+        name: Yandex IAM tokens should not be disclosed
+        message: Make sure this Yandex IAM token gets revoked, changed, and removed from the code.
+      detection:
+        pre:
+          include:
+            content:
+              - t1.
+        matching:
+          pattern: "\\b(t1\\.[\\w-]+[=]{0,2}\\.[\\w-]{86}[=]{0,2})\\b"
```

Comment:
To limit FPs, I would put a minimal amount of characters on the first `[\\w-]+`. From the example 70 or 80 might be safe enough, but you know better what this threshold should be.

Reply:
The first part can be as small as 1 character.

Reply:
Oh okay, nevermind this then.